web/html/forms-iv/formenctype.md

```diff
@@ -2,20 +2,26 @@
 author: Stefan-Stojanovic
 type: normal
 category: must-know
-links: null
+links:
+  - >-
+    [MDN enctype and formenctype](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes_for_form_submission){documentation}
 ---
 
-# The `formenctype` Attribute
+# enctype & formenctype Attributes
 
 
 ---
 
 ## Content
 
 The HTML `formenctype` input form attribute is used to specify how form-data should be encoded before it's sent to the server.
-It is only used for buttons and images with `type="submit"`.
+It is only used for buttons with `type="submit"` and `type="image"`.
 
-If the attribute is not set, the default value is set to `application/x-www-form-urlencoded`.
+An equivalent attribute is called `enctype`. It has the same functionality, except it is used on the form instead of the two mentioned button types.
```

Comment:
what do you think about this?
```suggestion
You could also use the `enctype` attribute on the `form` element. This way, you can decide the encoding type for the entire form, instead of specifying it for every `input`.
```